com.unity.formats.fbx/Documentation~/integration.md

```diff
@@ -121,13 +120,62 @@ __File__ > __Unity__ > __Export Model Only__ exports all Models in the selected
 
-> **NOTE:** If no animation file with the **@** notation has been imported, then this option has no effect. The workaround for exporting a new animation is to first import an empty FBX file with the **@** notation (*{model}@anim.fbx*), so that the export set is configured correctly.
-
 In order to export objects from the desired export set, you can select one or more objects in the set or the set itself. In either case, the FBX Exporter exports the entire contents of the set.
 
 If you select multiple sets or objects from multiple sets, then the FBX Exporter exports each set to its respective file defined in the attributes of the set.
 
+### Create a New Export Set in Maya
+
+In order to export objects that are not part of an existing export set, follow these steps to prepare an export set.
+First select the desired objects for export. Next, select one of the following menu items:
+
+![](images/FBXExporter_MayaUnityMenuItems.png)
+
+__File__ > __Unity__ > __Create Export Set__
+
+__File__ > __Unity__ > __Export__ 
+
+Select either of these options to create an export set with settings for exporting animation and model FBX files.
+
+__File__ > __Unity__ > __Export Model Only__
+
+__File__ > __Unity__ > __Export Animation Only__ 
+
+Select Model Only or Animation Only export to setup an export set for exporting a model or animation file.
+
+After selecting one of the above menu items, the following dialog will appear to set the options for the export set:
+
+![](images/FBXExporter_MayaCreateExportSetDialog.png)
+
+| Property:                     | Function:                                                    |
+| :---------------------------- | :----------------------------------------------------------- |
+| __Namespace__              | Select the namespace to add all the objects in the set to. Will not modify namespace of objects already in the desired namespace. |
+| __Model File Path__        | The file path to export to when calling __File__ > __Unity__ > __Export__ and __File__ > __Unity__ > __Export Model Only__. |
+| __Model File Name__        | The file name to export to when calling __File__ > __Unity__ > __Export__ and __File__ > __Unity__ > __Export Model Only__. |
+| __Anim File Path__         | The file path to export to when calling __File__ > __Unity__ > __Export Animation Only__. |
+| __Anim File Name__         | The file name to export to when calling __File__ > __Unity__ > __Export Animation Only__. |
+| __Strip Namespaces on Export__  | Check this option to automatically strip the most common namespace in the set on export (e.g. if most objects are in namespace `model:`, then `model:` will be stripped on export). |
+| __Create Set And Export__  | Creates the set and immediately exports selection. |
+| __Create Set__             | Creates set without exporting. |
+| __Cancel__                | Cancels set creation and closes dialog. |
```

Comment:
does it matter how many spaces there are before the `|` exactly? because you're not 100% consistent

Reply:
It doesn't matter when displayed in the webpage. I was trying to make them align somewhat to be easier to read when editing the file